packages/plugins/src/files/gridfs/gridfs-adapter.ts

```diff
@@ -29,15 +29,15 @@ export const GridFSAdapter: IFileAdapter = {
 
   ...FileAdapter,
 
-  async createSignedURL(directoryName, fileName) {
+  async createSignedURL(directoryName, fileName, context, isPrivate) {
```

Comment:
createSignedURL is used for uploading stuff, the expiry date hashing is therefore already there (in buildHashedFilename) so i think there is no changed needed here?